example-model/src/main/java/com/appunite/rx/example/model/dao/PostsDao.java

```diff
@@ -151,6 +173,16 @@ public PostDao postDao(@Nonnull final String id) {
         return refreshSubject;
     }
 
+    @Nonnull
+    public Observable<ResponseOrError<Response>> getPostSuccesObserver() {
```

Comment:
no "get" just "postSuccessObserver()" we don't use "get" when everything is final in class